LoadTests/src/test/scala/default/RecordingSimulation.scala

```diff
@@ -33,7 +33,7 @@ class RecordingSimulation extends Simulation {
 
 	val headers_3 = Map(
 		"Accept" -> "text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8",
-		"Origin" -> "https://wa-t1pp-gpg.azurewebsites.net",
+		"Origin" -> "https://gender-pay-gap-loadtest.london.cloudapps.digital",
```

Comment:
I'm a bit sad this is checked in to source control (and isn't e.g. environment config) but whatever